### PR TITLE
Use openAPI V3

### PR DESCRIPTION
**Pull request by raplemie**
# Swaggergen cli v3

Switched to use the V3 of the documents, mainly in prevision of FileShare section, where storage API properly uses the return values Folder | File that was not available in V2 (so GetFolderAndFilesForFolder returned "any", which is not convenient...)

Updated names to include API (StorageFile => StorageFileSynchronizationAPI) while it is longer, it will reduce name clashing when importing those types, these may become more and more likely as more API's are used...

(This also fixes Storage use of "File", as this model is now "FileStorageAPI"...)

### CODE REVIEW

File: packages/app/src/routers/SynchronizationRouter/components/BridgeIcon.tsx [hunk 1]
```diff
@@ -16,19 +16,19 @@ import { ReactComponent as SvgNwd } from "../../../svg/unknowndocument.svg";
 import "./BridgeIcon.scss";
 
 interface BridgeIconProps {
-  bridgeType?: IModelBridgeType;
+  bridgeType?: ConnectorTypeSynchronizationAPI;
```
Comment (aruniverse) [inline packages/app/src/routers/SynchronizationRouter/components/BridgeIcon.tsx:19]: might want to replace all occurrences of the term "bridge" for "connector" in the repo

Comment (raplemie) [reply packages/app/src/routers/SynchronizationRouter/components/BridgeIcon.tsx:19]: Will do in a separate PR, I dont want to obscure this one more than it is.

Comment (raplemie) [reply packages/app/src/routers/SynchronizationRouter/components/BridgeIcon.tsx:19]: Hmm, might as well change it now, I think I already touch all the lines where this term is used...

Comment (raplemie) [reply packages/app/src/routers/SynchronizationRouter/components/BridgeIcon.tsx]: Updated

**Review by kckst8 — APPROVED** [review]

**Pull request merged**